sysutils/osinfo-db/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                osinfo-db
+version             20190304
+platforms           darwin
+categories          sysutils gnome
+license             LGPL-2.1+
+maintainers         nomaintainer
+
+description         Libosinfo database files
+long_description    ${description}
+
+homepage            https://libosinfo.org
+master_sites        https://releases.pagure.org/libosinfo/
+
+checksums           rmd160  7c1394e42ee5372b92294cf2d7fda713a08abb7d \
+                    sha256  0e31c3f19f2f897dd2ba08d8656fe0bc4abc24f9c1558aff46bb9e9858cbbdb6 \
+                    size    83024
+
+depends_build       port:osinfo-db-tools
+
+use_xz             yes
+use_configure      no
+
+extract {}
+
+build {}
```

Comment:
So this is basically a database file? If so, add the following line below `maintainers`:

`supported_archs noarch
`

Reply:
I guess it is. I've made the change.